ng/src/web/components/table/stripedtable.js

```diff
@@ -43,7 +44,42 @@ const StripedTable = glamorous(Table)({
       background: '#DDDDDD',
     },
   },
-});
+},
+  ({doubleRow}) => {
+    if (doubleRow) {
+      return {
+        // FIXME when details are toggled, coloring is scrambled up, due to
+        // adding just one more tr and messing with the groups of two
+        '@media screen': {
+          // overwrite standard striped coloring
+          ['& > tbody:nth-of-type(even), ' +
+          '& > tbody:only-of-type > tr:nth-of-type(even)']: {
+            background: '#FFFFFF',
+          },
+          // Think in groups of 4 and color every two rows identical
+          ['& > tbody:nth-of-type(even), ' +
+          '& > tbody:only-of-type > tr:nth-of-type(4n), ' +
+          '& > tbody:only-of-type > tr:nth-of-type(4n-1)']: {
+            background: '#EEEEEE',
+          },
+          ['& > tbody:not(:only-of-type):hover, ' +
+          '& > tbody:only-of-type > tr:hover,' +
+          '& > tbody:only-of-type > tr:hover + tr']: {
+            background: '#DDDDDD',
+          },
+          // FIXME hovering groups don't fit when mouse over even rows
+          ['& > tbody:only-of-type > tr:nth-of-type(even):hover,' +
```

Comment:
The coloring of the grouped rows only works, when the first column is hovered, that means effectively hovering every even row. When hovering an odd row (i.e. the descriptions) the grouping breaks and the next even row is colored together with the currently hovered odd row.
Without the surrounding TableBody I can't find a solution to group an even row together with the PREVIOUS one. Any suggestions are highly appreciated!